homeassistant/components/xiaomi_miio/sensor.py

```diff
@@ -377,10 +395,23 @@ class XiaomiMiioSensorDescription(SensorEntityDescription):
 )
 PURIFIER_MIOT_SENSORS = (
     ATTR_FILTER_LIFE_REMAINING,
+    ATTR_FILTER_LEFT_TIME,
```

Comment:
Is `ATTR_FILTER_LEFT_TIME` added intentionally to the existing `PURIFIER_MIOT_SENSORS` device?

Reply:
Updated to be only for Purifier 4 and 4 Pro